README.md

```diff
@@ -116,3 +120,74 @@ Modify `nrf802154_sniffer.py`:
     ```python
     #DLT='802.15.4'
     ```
+
+## Troubleshooting
+
+#### Sniffer capture hangs when another Wireshark process is started with nRF BLE sniffer extcap
+This issue is caused by BLE sniffer extcap, which discovers connected sniffers during Wireshark startup by actively sending data to all serial ports.
+Because Linux applications primarily use advisory locking, there is nothing stopping other applications from opening and writing data to the serial port, causing unexpected behaviour.
+Currently, the only way to avoid this issue is to disable the nRF BLE sniffer extcap by removing the executable flag:
+```bash
+sudo chmod -x <extcap_path>/nrf_sniffer.py
+```
+
+#### Sniffer capture hangs with `ModemManager` service enabled on Linux
+On some occasions the `ModemManager` may send AT commands to the sniffer. To avoid it do one of the following:
+* Disable `ModemManager` service
+```bash
+sudo systemctl stop ModemManager.service
+sudo systemctl disable ModemManager.service
+```
+* Create `udev` rule (if `ModemManager` runs on `DEFAULT` or `PARANOID` policy):
+    1. Create new file `/etc/udev/rules.d/99-mm-blacklist.rules` and write the below configuration:
+        ```
+        ACTION!="add", SUBSYSTEM!="usb_device", GOTO="mm_blacklist_rules_end"
+
+        ATTR{idProduct}=="154a", ATTR{idVendor}=="1915", ENV{ID_MM_DEVICE_IGNORE}="1"
+
+        LABEL="mm_blacklist_rules_end"
+        ```
+    2. Apply the new `udev` rules:
+        ```bash
+        udevadm trigger
+        ```
+    3. Verify that the settings have been successfully applied:
+        ```bash
+        udevadm info -q property -n /dev/ttyACMx
+        ```
+        Find the following values:
+        ```
+        ID_MM_CANDIDATE=0
+        ID_MM_DEVICE_IGNORE=1
+        ```
+    4. Restart the `ModemManager`:
+        ```bash
+        sudo systemctl restart ModemManager
+        ```
+    __Note__: The above steps will not work if `STRICT` policy of the `ModemManager` is used.
+
+#### Sniffer interface does not appear in the Wireshark
+##### On Ubuntu Linux
+You may not have sufficient permissions to access the serial device.
+On Ubuntu you have to add the user to the `dialout` group.
+To remedy this execute the following command:
+```bash
+sudo usermod -a -G dialout [user]
+```
+You may also have to add the user to the `wireshark` user group to start the capture:
+```bash
+sudo usermod -a -G wireshark [user]
+```
+Ensure that `pySerial` module was installed by the correct Python interpreter used by the extcap script.
```

Comment:
I'm a bit confused: is this a 3rd step in a procedure started at line 173? If not, why should I ensure that the module is installed?

Reply:
Frequently it is necessary to install more than one Python interpreter. Usually it is the Python 2.7 and Python 3.x. The script supports both. Extra care needs to be taken to ensure that the Python interpreter used for running the script is the one with the packages installed.
Fortunately, next year Python 2.7 will reach EOL and this will become less of a problem.